### ISSUE TITLE
XDG support

### ISSUE THREAD

**Issue by polyzen** [nuxeh/url-bot-rs#19]
XDG support may want to be considered. For example I'm using `~/.config/url-bot/config.toml` and `~/.local/share/url-bot/history.db`. For a default DB path to work, perhaps enabling that can be moved to another flag like `--history`.

**Comment by nuxeh** [comment]
Yes, indeed, totally agree. I was actually doing a few things along the same lines a little while ago, the idea was to have default search paths, for cases where a path isn't specified on the command line, for the conf or the db. XDG is an interesting suggestion.

**Comment by polyzen** [comment]
> For a default DB path to work, perhaps enabling that can be moved to another flag like --history.

Misunderstanding here, I thought the previous post feature was completely disabled unless a DB file was used.

**Comment by nuxeh** [comment]
A few crates:
https://crates.io/crates/dirs
https://crates.io/crates/directories
https://crates.io/crates/app_dirs
